src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+  },
+});
```

Comment:
>Includes an array of categories (initial state: empty array)
Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string)

- You have done great so far :+1: You set up redux correctly, but there are some requirements you missed that need to be implemented before moving to the next milestone. :+1: :100: 
- [ ] Looking at the project requirement above, you can see that the category needs to have a reducer that checks status and return a string of `Under construction`. currently, you do not have any reducer implemented, I'll suggest you add a new reducer `checkStatus` and them implement the functionality. :+1: :wave:

Reply:
Thanks, I appreciate your professional review.